pkg/systemd/logs.scss

```diff
@@ -35,6 +35,16 @@
 }
 
 #log-details {
+    .pf-c-card__header {
```

Comment:
This should be done from JS, the link can be put in the CardTitle, instead of the CardAction and this is not needed.

Also this breaks other views like:
![Screen Shot 2021-11-17 at 18 33 07](https://user-images.githubusercontent.com/14921356/142252449-12e236a7-6abe-43cb-9c32-1679121273d1.png)

Reply:
So I removed this comimit, as in this PR, I also changed that card title to be the unit name, and not the syslog idendifier. So the initial issue is solved.

Reply:
Yeah, this works. We should make the link more obvious too, by not having it so far away. People had issues finding it. But it can be handled in another PR. What's here is a huge improvement and I'm happy you did this (and so quickly)!